seqr/views/utils/file_utils_2_3_tests.py

```diff
@@ -52,10 +56,31 @@ def _mock_cell(value):
 MOCK_EXCEL_SHEET = mock.MagicMock()
 MOCK_EXCEL_SHEET.iter_rows.return_value = [[_mock_cell(cell) for cell in row] for row in PARSED_DATA]
 
+def _convert_int(s):
+    try:
+        ret_value = int(s)
+    except Exception:
+        return s
+    return ret_value
+
 
 class FileUtilsTest(AuthenticationTestCase):
     fixtures = ['users']
 
+    def setUp(self):
+        # Test uploading with xlsx data
+        wb = xl.Workbook()
+        ws = wb[wb.sheetnames[0]]
+        ws['A1'], ws['B1'], ws['C1'] = map(_convert_int, PARSED_DATA[0])
+        ws['A2'], ws['B2'], ws['C2'] = map(_convert_int, PARSED_DATA[1])
+        ws['A3'], ws['B3'] = map(_convert_int, PARSED_DATA[2][:2])
```

Comment:
Given that this is a small file, this would be much more readable if you just wrote the data you want to write here, instead of doing a mapping on `PARSED_DATA`  (i.e. `ws[1] = [0, 'NA19678', '']...`)

Reply:
Okay